help/editorconfig.md

```diff
@@ -1,33 +1,22 @@
-editorconfig-micro
-==================
+editorconfig
+============
 
-[EditorConfig][] helps developers define and maintain
+http://editorconfig.org helps developers define and maintain
 consistent coding styles between different editors and IDEs.
 This is the EditorConfig plugin for the `micro` editor.
 
 This plugin requires an editorconfig core executable to be installed.
-For example, download the [EditorConfig C Core][] and follow the instructions in
-the README and INSTALL files to install it.
+https://github.com/editorconfig/editorconfig-core-c
 
 
 Usage
 -----
 
-Once installed, this plugin will automatically execute `editorconfig.getApplyProperties`
-on files when they are opened or saved.
+Once installed, this plugin will automatically use the editorconfig core to
+obtain the desired editorconfig properties based on the file, and apply the
+corresponding micro options on both open (necessary for things like tabstospaces)
+and save (necessary for things like rmtrailingws).
 
-You can also explicitly use the `editorconfig` command in command mode, or bind it to
```

Comment:
@10sr In case you were wondering, these two other ways of executing the plugin don't work anymore. According to debug logs, it is being run, but things like rmtrailingws aren't actually modifying the buffer for some reason. But I opted to just remove them from documentation since I feel like no one uses them. `"editorconfig.getApplyProperties"` doesn't turn up in GitHub-wide search in peoples' dotfiles, and I can't imagine anyone typing out `editorconfig` when they can just instead save the file.

Reply:
This was defined because originally this plugin did not run getApplyProperties on save.
Now it does, so OK to remove this command 👍